sig-cli/annual-report-2022.md

```diff
@@ -4,79 +4,87 @@
+
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - [Open untriaged issues and PRs](https://cli.triage.k8s.io/s/kubectl).
+   - New reviewers and approvers added.
+     - Last year we added 1 reviewer and 1 approver.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - No this document is out of date and difficult to maintain. We need to update it and model it after some other groups.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We recently did a reviewer mentorship cohort. Unclear if we would do another. The group consisted mostly of students not very experienced with Golang.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+    - The leads all represent different companies.
+    - Contributors from 28 companies, 9 of which had 10+ contributions.
+      - [Data](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20CLI&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=now-1y%2Fy&to=now-1y%2Fy).
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - Kustomize is in need of new maintainers. Ideally folks that depend on it daily and able to jump right in.
+   - kubectl is a large tool and we need new folks to stick around and learn its different pieces.
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Primary slack channel member count: 2361
+- Primary mailing list member count: 425
+- Primary meeting attendee count (estimated, if needed): 10+
+- Primary meeting participant count (estimated, if needed): 6
+- Unique reviewers for SIG-owned packages: 32
```

Comment:
@soltysh you generated these numbers with https://github.com/MadhavJivrajani/owners-counts but they are suspiciously higher than what we reported last year.

https://github.com/kubernetes/community/blob/a055fba9ddab59a10db8b4149c9bea8d5a3a596b/sig-cli/annual-report-2021.md?plain=1#L81-L82

Reply:
Fair point, I haven't checked the output of the tool, not didn't compare that with last year. Probably worth double checking and reporting to @MadhavJivrajani a problem. I'm pretty sure we have more approvers, but you're right the delta is too big.